src/components/Transactions/transactionDetail.test.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { dummyTransactionsData } from 'stories/utils/mocks';
+import TransactionDetail from './TransactionDetail';
+import { useReceiptData } from 'hooks/useReceiptData';
+
+jest.mock('hooks/useReceiptData');
+jest.mock('react-i18next', () => ({
+  // this mock makes sure any components using the translate hook can use it without a warning being shown
+  useTranslation: () => {
+    return {
+      t: (str: string) => str,
+      i18n: {
+        changeLanguage: () => new Promise(() => {})
+      }
+    };
+  },
+  initReactI18next: {
+    type: '3rdParty',
+    init: () => {}
+  }
+}));
```

Comment:
Can we make this mock available by default in every test?

Should be possibile moving this code in another file and referencing it by the `setupFile` property in `jest.config.js` file.

https://stackoverflow.com/questions/49685265/jest-mock-user-module-in-all-test-files